### PR TITLE
changed to sdk.Uint

### PR DESCRIPTION
**Pull request by ErinHales**

### CODE REVIEW

**Comment by prestwich** [comment]
fix tests and merge conflicts?
```
--- FAIL: TestBTCUtils (0.01s)
    --- FAIL: TestBTCUtils/TestParseHeader (0.00s)
        suite.go:61: test panicked: interface conversion: interface {} is []uint8, not uint64
    --- FAIL: TestBTCUtils/TestParseInput (0.00s)
        validate_spv_test.go:64: 
            	Error Trace:	validate_spv_test.go:64
            	Error:      	Not equal: 
            	            	expected: 2
            	            	actual  : 1
            	Test:       	TestBTCUtils/TestParseInput
    --- FAIL: TestBTCUtils/TestProve (0.00s)
        suite.go:61: test panicked: interface conversion: interface {} is bool, not []uint8
    --- FAIL: TestBTCUtils/TestValidateHeaderChain (0.00s)
        suite.go:61: test panicked: interface conversion: interface {} is int, not uint64
    --- FAIL: TestBTCUtils/TestValidateHeaderWork (0.00s)
        suite.go:61: test panicked: interface conversion: interface {} is int, not uint64
```

**Comment by ErinHales** [comment]
Ok, those errors are resolved now, should be good to go.

File: golang/pkg/btcspv/bitcoin_spv.go [hunk 1]
```diff
@@ -347,28 +341,28 @@ func ExtractMerkleRootBE(header []byte) []byte {
 }
 
 // ExtractTarget returns the target from a given block hedaer
-func ExtractTarget(header []byte) sdk.Int {
+func ExtractTarget(header []byte) sdk.Uint {
 	// nBits encoding. 3 byte mantissa, 1 byte exponent
 	m := header[72:75]
 	e := sdk.NewInt(int64(header[75]))
```
Comment (prestwich) [inline golang/pkg/btcspv/bitcoin_spv.go]: should be `NewUint`?

Comment (ErinHales) [reply golang/pkg/btcspv/bitcoin_spv.go]: Actually, if I remember right, there was one place where it needed to be int

Comment (prestwich) [reply golang/pkg/btcspv/bitcoin_spv.go]: Oh, got it. It's because of the weird integer wrapping thing they do

**Pull request merged**